proto/cfeairdrop/account.proto

```diff
@@ -7,24 +7,29 @@ import "cosmos/vesting/v1beta1/vesting.proto";
 
```

Comment:
move AirdropVestingAccount to vesting module and rename RepeatedContinuousVestingAccount